elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/ElytronOidcExtension.java

```diff
@@ -62,7 +64,8 @@ public void initialize(ExtensionContext context) {
 
     public void initializeParsers(ExtensionParsingContext context) {
         context.setSubsystemXmlMapping(SUBSYSTEM_NAME, ElytronOidcSubsystemParser_1_0.NAMESPACE_1_0, ELYTRON_OIDC_SUBSYSTEM_PARSER_1_0);
-        context.setSubsystemXmlMapping(SUBSYSTEM_NAME, ElytronOidcSubsystemParser_2_0.NAMESPACE_2_0, CURRENT_PARSER);
+        context.setSubsystemXmlMapping(SUBSYSTEM_NAME, ElytronOidcSubsystemParser_2_0.NAMESPACE_2_0, ELYTRON_OIDC_SUBSYSTEM_PARSER_2_0);
+        context.setSubsystemXmlMapping(SUBSYSTEM_NAME, ElytronOidcSubsystemParser_3_0.NAMESPACE_3_0, CURRENT_PARSER);
```

Comment:
This schema should not be registered if the process was started with --stability=DEFAULT
Consider refactoring your subsystem schemas/parsers using PersistentSubsystemSchema and leverage this method for parser registration: https://github.com/wildfly/wildfly-core/blob/main/controller/src/main/java/org/jboss/as/controller/parsing/ExtensionParsingContext.java#L82